openfe/tests/protocols/test_openmm_equil_rfe_protocols.py

```diff
@@ -786,3 +788,27 @@ def test_constraints(tyk2_xml, tyk2_reference_xml):
             assert a.get('p1') == b.get('p1')
             assert a.get('p2') == b.get('p2')
             assert float(a.get('d')) == pytest.approx(float(b.get('d')))
+
+
+@pytest.mark.slow
```

Comment:
As per my comments last week, can we please not have tests that run the integrator in the standard slow tests? I need to be able to dry run things regularly when I work on things but if I have to also run a simulation that adds a massive additional burden.

My take is that you should make this a "runintegrator" flag or similar. It's only really necessary because we assume that some folks will have platforms that can't run an integrator.

Reply:
I'll go back on this for now if we can make this more of a "minimal" test - i.e. a methane self transformation. Assuming a stable initial conformation, that should "always" be stable (since there are no transforming atoms happening)

Reply:
We should also enforce OPENMM_CPU_THREADS=1

Reply:
I've added a fixture for threads=1, it doesn't seem to make any difference for me locally however.  The tests take 20s/5s for CPU/CUDA respectively, I think this is in line/faster than other slow tests, so it's probably ok to not label them differently.

Reply:
Can you undo the env variable and check here? The gh actions CPUs are very very old

Reply:
Looks like it ran pretty quickly, we could just leave it in the non slow tests tbh